rpa-builder/6.7/modules/ROOT/pages/toolbox-mail-operations-mail-session-oauth.adoc

```diff
@@ -0,0 +1,142 @@
+
+== Wizard
+
+image:mail-session-outlook-oauth-wizard.png[The Mail Session (OAuth) Action Step Wizard, 60%, 60%]
+
+The wizard contains the following sections:
+
+* *Operation Mode*
+* *OAuth Authentication*
+* *Incoming Mail*
+* *Outgoing Mail*
+
+=== Operation Mode
+
+In *Operation Mode* you determine what happens and which area activates in the Mail Session. The
+following modes are available:
+
+* *Read and send mails*
++
+Enables you to configure the Incoming and Outgoing
+Mail areas so that you can read and send emails.
+* *Read mails from inbox*
++
+Enables editing for the Incoming Mail section so that you can only read emails.
++
+If you select the _Read mails from inbox_ mode, you can't use the Send Mail Action Step.
+* *Send mails via SMTP*
++
+Enables editing for the Outgoing Mail section so that you can only send emails.
++
+If you select the _Send mails via SMTP_ mode, you can't use the Read Mail Action Step in the Mail Session.
+
+=== OAuth Authentication
+
+The *OAuth Authentication* section enables you to create a set of tokens that don't rely on a user password to authenticate with the email server. After you create the refresh token, you can use it to repeatedly log in to the services without asking for the user's password again.
+
+Follow these steps to authenticate with your credentials:
+
+. Complete the *OAuth Authentication* configuration by specifying the required properties.
++
+Refer to <<properties, OAuth Authentication Settings>> for a description of each property.
+. Click *Authenticate*.
++
+This step starts the authentication process with the OAuth Host, which opens the login page in your default browser. 
++
+To enable using different accounts to log in, Mail Session (OAuth) clears credentials stored in browser cookies when you click *Authenticate*. 
+. In the OAuth Host's login page, complete the login process.
+. Close the browser.
+
+After completing the last step, the wizard shows that you are authenticated:
+image:email-outlook-authentication.png[The Logged in confirmation message in the authentication settings window, 50%, 50%]
+
+[[generate-oauth-token]]
+== Generating an OAuth Refresh Token
+
+Instead of authenticating via the *Mail Session (OAuth) wizard*, you can pin the refresh token if you obtain it externally. In this case, you still need to populate or pin the remaining fields.
+
+include::partial$generate-oauth-token-steps.adoc[]
+
+=== Incoming Mail
+
+In the *Incoming Mail* area, you can configure all the settings necessary to import emails from an email server. You can use the *Test Connection Button* to test whether the connection to the email server can be established.
+
+=== Outgoing Mail
+
+The *Outgoing Mail* area opens if you select the *Send mails* checkbox. To send emails, enter the SMTP email server settings here.
+
+Use the *Send test mail* button to check the connection to the SMTP server and send a test email to the email address specified in the *Sendtest mail to* field.
```

Comment:
To check the connection to the SMTP server and send a test email to the email address specified in the *Sendtest mail to* field, click *Send test mail*.